design/js/slider-handlers.js

```diff
@@ -0,0 +1,21 @@
+document.addEventListener('DOMContentLoaded', () => {
+    document.querySelectorAll('.weight').forEach(weightContainer => {
+        const slider = weightContainer.querySelector('input[type="range"]');
+        const minusButton = weightContainer.querySelector('.bi-dash-circle-fill');
+        const plusButton = weightContainer.querySelector('.bi-plus-circle-fill');
+
+        minusButton.addEventListener('click', () => {
+            if (slider.value > slider.min) {
+                slider.value = parseInt(slider.value) - 1;
+                slider.dispatchEvent(new Event('input')); // Trigger input event if needed
```

Comment:
Are there any technical reasons for this?

Reply:
Hi @cmllr,
Initially, I added the input event dispatch to ensure that any future event listeners tied to slider changes would be triggered consistently. However, since there are no current listeners, it's not necessary.

There are no specific technical reasons to keep the input event dispatch in the current implementation. I'll remove it to keep the code clean and efficient.
I'll update the PR accordingly. Thanks for your input!

Reply:
Hey, I made 2 new commits regarding adding `cursor: pointer` and removing unnecessary input event dispatch from slider handlers. Let me know if further changes are required!